tests/handle-tag.js

```diff
@@ -42,7 +42,7 @@ describe('Handle tag', () => {
         mockHandleTag.returns('');
         http.get('http://localhost:8080/template', (response) => {
             const request = mockHandleTag.args[0][0];
-            const tag = mockHandleTag.args[0][1];
+            const tag = mockHandleTag.args[0][3];
```

Comment:
can we keep the tag as arg1 .. otherwise it wont be backwards compatible.

Reply:
aaaaah
good point
yes